### PR TITLE
first functions

### PR DESCRIPTION
**Pull request by KarelVeldkamp**
i implemented the first functions of the app. (get song lyrics, get album lyrics, get artist network, get images)

### CODE REVIEW

File: musicApp/.Rproj.user/5ABFF95C/sources/per/t/C1CBF118-contents [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+  network = list(nodes = nodes, edges = edges)
+
+  return(network)
+}
+
+
+####################################################################################################
+# function that returns a dataframe containing the urls of artist images based on a vector of ids
+getImages <- function(artistIds)
```
Comment (SvdStappen) [inline musicApp/.Rproj.user/5ABFF95C/sources/per/t/C1CBF118-contents:69]: Here you use artistIds as input. But how will the user know these IDs? I see an example of Ids is obtained in the previous function. You could return this to the user, or you could choose to ask the user to provide artist names instead of IDs (and then look for the corresponding Ids in the function, just like in the first function). This might be more user friendly.

**Review by SvdStappen — COMMENTED** [review]
General tip: you can use cltr + shift + r in Rstudio to make subdivisions with titles, that you can also hide if preferred. This helps making the structure clearer.

You sometimes use the code package::function, but for most of the codes you call library on top of your file. Would be good to be consistent in this, and then preferably use the package::function method. In addition, you should add the necessary packages to your description file (see here, header Dependencies: http://r-pkgs.had.co.nz/description.html for how to do this).

I also added some comments to a specific code line.

**Review by mrooyakkers — COMMENTED** [review]
Your code looks great and I am excited to eventually use this package! My only critique is that your annotation/comments are a little hard to follow as a R beginner - making it (sometimes) difficult to determine what your function is doing.
